server/routes/juror-management/juror-record/juror-record.controller.js

```diff
@@ -241,7 +241,7 @@
       const promiseArr = [];
 
       if (req.query.loc_code) {
-        req.session.locCode = req.query.loc_code;
+        req.session.authentication.locCode = req.query.loc_code;
```

Comment:
we should not mutate this... it is the data that gets decoded from the token 🙃  and changing it will break the app in other places....

Reply:
...